docs/tutorial/commandlineinterface.rst

```diff
@@ -290,23 +287,23 @@ either of these commands:
 
 .. code:: bash
 
-   lobsterpy calc-description -help
+   lobsterpy create-inputs --help
 
 .. code:: bash
 
-   lobsterpy create-inputs --help
+   lobsterpy description --help
```

Comment:
Oh, I think you mixed something up here, @naik-aakash . Didn't see it at first

Reply:
It is fine, just re-orderdered the commands

Reply:
Okay